lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -261,8 +261,8 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
       kind: ui.PointerDeviceKind.mouse,
       signalKind: ui.PointerSignalKind.scroll,
       device: _mouseDeviceId,
-      physicalX: event.client.x * ui.window.devicePixelRatio,
-      physicalY: event.client.y * ui.window.devicePixelRatio,
+      physicalX: event.client.x.toDouble() * ui.window.devicePixelRatio,
```

Comment:
We used to have type casts for all of these. They got removed in [this PR](https://github.com/flutter/engine/pull/20758) by @a-siva after some changes in static type rules for null-safe numbers?

Reply:
yes num as double not right. .toDouble() will correctly convert (most likely this is noop after dart2js since everything is a num after all).